managegroup.php

```diff
@@ -239,9 +239,10 @@
 
 	$plugins->run_hooks("managegroup_do_manageusers_start");
 
-	if(is_array($mybb->get_input('removeuser', MyBB::INPUT_ARRAY)))
+	$users = $mybb->get_input('removeuser', MyBB::INPUT_ARRAY);
```

Comment:
This should probably be added before running the `managegroup_do_manageusers_start` hook.

Reply:
```suggestion
```